### PR TITLE
Chore/add new aggregation app

### PR DESCRIPTION
**Pull request by nemo83**

### CODE REVIEW

File: deploy/cf-ledger-sync/templates/ledger-sync-deployment.yaml [hunk 1]
```diff
@@ -92,10 +92,8 @@ spec:
                 secretKeyRef:
                   name: postgres-secrets
                   key: POSTGRES_DB
-            - name: SCHEMA
```
Comment (nemo83) [inline deploy/cf-ledger-sync/templates/ledger-sync-deployment.yaml:95]: @rcmorano out of curiosity, why did you remove it?

Comment (rcmorano) [reply deploy/cf-ledger-sync/templates/ledger-sync-deployment.yaml:95]: Because it was removed as an independent variable and now it's set as part of the jdbc url, check commit [31861a03cea50b11f23773a6e11bae2a7204df03](https://github.com/cardano-foundation/cf-ledger-sync/commit/31861a03cea50b11f23773a6e11bae2a7204df03)
